packages/axios-extension/src/abap/message.ts

```diff
@@ -91,13 +102,29 @@ export function prettyPrintMessage({ msg, log, host }: { msg: string; log: Logge
  * @param root0.host hostname
  * @param root0.path path
  * @param root0.log log
+ * @param root0.isDest destination
  */
-function logFullURL({ host, path, log }: { host: string; path?: string; log: Logger }): void {
+function logFullURL({
+    host,
+    path,
+    log,
+    isDest = false
+}: {
+    host: string;
+    path?: string;
+    log: Logger;
+    isDest?: boolean;
+}): void {
     if (host && path) {
         const base = new URL(host).origin; // We only care for the origin value
         // Add this instruction to the user because of this bug in VS Code: https://github.com/microsoft/vscode/issues/144898
         // It undoes the encoding of special characters in the URL sent to the browser
         log.info('Please copy/paste this URL in a browser for more details:');
+        if (isDest) {
+            log.info(
+                '(Note: As the destination uses a virtual host, you will need to replace the host in the URL with the internal host that is configured with your SAP Cloud Connector)'
```

Comment:
@theStiggySpd this info message needs to be changed. Yes, we know its a destination but the URL defined in the destination properties, is not always a virtual host. Its only a virtual host when its on-premise. A suggestion might be;
```
(Note: You will need to replace the host in the URL with the internal host, if your destination is configured using an On-Premise SAP Cloud Connector)
```
In alot of cases, the URL defined is a direct mapping to the internal host mapping defined in the SCC but there is not always valid.

Reply:
Adjusted the message, if @theStiggySpd approves

Reply:
Looks good @longieirl @mikicvi-SAP